docs/product/performance/filters-display/widgets.mdx

```diff
@@ -110,3 +110,11 @@ Shows transactions with slow cumulative resource spans. These show frontend tran
 This widget is available in the [Frontend](/product/performance/filters-display/#frontend) tab.
 
 ![Performance Widgets - Slow Resource Ops](./img/widgets_slow_resource_ops.png)
+
+## Most Time-Consuming Domains
```

Comment:
Can we move this section to after "most time-consuming resources"? I think this would just make the right side nav easier to skim.

Reply:
Done!